### PR TITLE
Modifications

### PR DESCRIPTION
**Pull request by TuomasBorman**
Bray-curtis is usually applied to relative abundances

### CODE REVIEW

**Comment by TuomasBorman** [comment]
https://github.com/microbiome/OMA/issues/170

File: 98_exercises.Rmd [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+2. Load experimental dataset from mia.
+3. What are the dimensions? (How many samples there are, and how many taxa in each taxonomic rank)?
+4. Calculate library size.
+5. Summarize sample metadata variables. (How many age groups, how they are distributed? 0%, 25%, 50%, 75%, and 100% quantiles of library size?)
+6. Create two histograms. Another shows the distribution of absolute counts, another shows how CLR transformed values are distributed.
+7. Visualize how relative abundances are distributed between taxa in samples.
+8. Calculate Shannon diversity index. Is it significantly different between vegan and mixed diet?
+
+Functions that might be useful: nrow, ncol, dim, summary, table, quantile, unique, transformSamples, ggplot, wilcox.test, addPerCellQC, agglomerateByRank, estimateDiversity, plotAbundance
+
```
Comment (antagomir) [inline 98_exercises.Rmd:38]: Suggestions if situation allows. Here and in other tasks.

- add hyperlinks to function manpages.. or highlight the code
- add cross-links to relevant chapters in OMA

File: 98_exercises.Rmd [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+## Basics of community composition
+
+1. Please install the latest development version of mia from GitHub.
+2. Load experimental dataset from mia.
+3. Create a PCoA with Aitchison dissimilarities. How much coordinate 1 explains the differences? How about coordinate 2?
+4. Create dbRDA with Bray-Curtis dissimilarities on relative abundances. Use PERMANOVA. Can differences between samples be explained with variables of sample meta data? 
+5. Analyze diets' association on beta diversity. Calculate dbRDA and then PERMANOVA. Visualize coefficients. Which taxa's abundances differ the most between samples? 
+6. Interpret your results. Is there association between community composition and location? What are those taxa that differ the most; find information from literature.
+
+Functions that might be useful: runMDS, runRDA, anova.cca, transformSamples, agglomerateByRank, ggplot, plotReducedDim
```
Comment (antagomir) [inline 98_exercises.Rmd:58]: Useful functions could be potentially listed after each numbered task in parentheses (but good like this already)

File: 98_exercises.Rmd [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Exercises"
+output: html_notebook
+---
+
+Here you can find assignments on different topics. 
+
+TIPS:
```
Comment (antagomir) [inline 98_exercises.Rmd:8]: This part could contain link to demo data chapter, for example data(?)

**Review by antagomir — COMMENTED** [review]
Looks good!

File: 98_exercises.Rmd (hunk 2 above)
Comment (TuomasBorman) [reply 98_exercises.Rmd:58]: I'm wandering if that is too easy. Googling and searching from OMA could be part of the task (like in real life)

referring also hyperlinks

**Pull request merged**